services/core-api/tests/reports/resource/test_mine_reports_resource.py

```diff
@@ -61,28 +60,28 @@ def test_get_code_required_reports_for_mine(test_client, db_session, auth_header
 
     get_resp = test_client.get(
         f'/mines/{mine.mine_guid}/reports?mine_reports_type=CRR&due_date_start={start_date.strftime("%Y-%m-%d")}&due_date_end={end_date.strftime("%Y-%m-%d")}',
-        headers=auth_headers['full_auth_header']
-    )
+        headers=auth_headers['full_auth_header'])
     get_data = json.loads(get_resp.data.decode())
     assert get_resp.status_code == 200
 
     for report in get_data['records']:
         received_date = datetime.strptime(report['received_date'], '%Y-%m-%d')
 
-        assert (start_date <= received_date.date())
-        assert (received_date.date() <= end_date)
+        assert (start_date <= received_date)
+        assert (received_date <= end_date)
```

Comment:
wait, did you fix our extremely flaky python test??

Reply:
I'm trying too, but i think so?

the TODAY generator in the factory file is being very odd....